lib/galaxy/tools/toolbox/watcher.py

```diff
@@ -14,6 +14,10 @@
     PollingObserver = None
     can_watch = False
 
+from galaxy.queue_worker import (
+    reload_data_managers,
+    reload_toolbox,
+)
```

Comment:
Nothing in ``galaxy.tools.toolbox`` should depend on ``galaxy.queue_worker`` IMO. The former module is meant to stand-alone from the app and models - hence the subclassing of ToolBox from AbstractToolBox and the inclusion on that module in galaxy-lib. Can you refactor this config watcher stuff into say ``galaxy.tools.watchers`` or ``galaxy.webapps.galaxy.config_watchers`` perhaps.